### PR TITLE
fix applying viewId instead of placeholderName

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

File: modules/web/js/ballerina/components/transform-render.js [hunk 1]
```diff
@@ -968,7 +968,7 @@ addComplexParameter(parentId, struct) {
         const xTargetPointer = 0;
         let yTargetPointer = 0;
         const functionGap = 30;
-        const svgLines = $('#' + self.placeHolderName + '> svg');
+        const svgLines = $('#' + self.viewId + '> svg');
```
Comment (pahans) [inline modules/web/js/ballerina/components/transform-render.js]: you can use a es6 string template string here. ``` $(`#${self.viewId}> svg`);```

Comment (hemikak) [inline modules/web/js/ballerina/components/transform-render.js]: I think @roonyh suggested to use 'this.container.find' instead of $

Comment (roonyh) [inline modules/web/js/ballerina/components/transform-render.js]: yep

Comment (roonyh) [inline modules/web/js/ballerina/components/transform-render.js]: We have used a different solution. The viewId in this case should differentiate svgLines in separate tabs. So $ is fine here. :)

**Pull request merged**